pandas/io/html.py

```diff
@@ -236,104 +239,104 @@ def _text_getter(self, obj):
         """
         raise AbstractMethodError(self)
 
-    def _parse_td(self, obj):
-        """Return the td elements from a row element.
+    def _equals_tag(self, obj, tag):
+        """Returns whether an individual DOM node matches a tag
 
         Parameters
         ----------
-        obj : node-like
+        obj : a DOM node.
+
+        tag : str
+            Tag to be checked for equality
 
         Returns
         -------
-        columns : list of node-like
-            These are the elements of each row, i.e., the columns.
+        boolean
+            boolean indicating if the object is equal to tag 'tag'
```

Comment:
Write:

~~~python
is_tag_equal : boolean
    boolean indicating if the object is equal to tag 'tag'
~~~